rpc-rs/Cargo.toml

```diff
@@ -46,7 +46,7 @@ num-bigint = { version = "0.4", optional = true }
 bigdecimal = { version = "0.3", optional = true }
```

Comment:
Don't forget the version bump for rpc. This is also a minor bump because we are making a breaking change to the public api of `Transport`. Shouldn't impact anyone, but technically breaking